SymHop/src/SymHop.cpp

```diff
@@ -365,7 +365,7 @@ void Expression::commonConstructorCode(QStringList symbols, bool &ok, const Expr
         int start=0;        //Start index of each symbol
         for(int i=0; i<str.size(); ++i)
         {
-            if(!var && parBal==0 && (str.at(i).isLetterOrNumber() || str.at(i) == '_' || str.at(i) == '-' || str.at(i) == '.' || str.at(i) == ':')) //New variable or function string
+            if(!var && parBal==0 && (str.at(i).isLetterOrNumber() || str.at(i) == '_' || str.at(i) == '-' || str.at(i) == '.' || str.at(i) == ':' || str.at(i) == '"')) //New variable or function string
             {
```

Comment:
It is unclear to me why these changes are needed

Reply:
It is needed so that string parameters can be set through HCOM. Consider the following commands:
> chpa Mass.B.y self.m
> chpa Mass.B.y "self.m"
The first will set Mass.B.y to the evaluated value of "self.m". The second will set Mass.B.y to reference "self.m". But this requires SymHop to support quoted strings.